src/inausoft.netCLI.Tests/RootCommandHandlerTests.cs

```diff
@@ -76,10 +73,11 @@ public void RootCommandHandler_RunsOnlyProperCommandHandler_WhenMultipleCommandH
 
             var mockCommand1Handler = new MockCommand1Handler();
             var mockCommand2Handler = new MockCommand2Handler();
-            var rootCommandHanlder = new RootCommandHandler(new ICommandHandler[] { mockCommand2Handler, mockCommand1Handler });
+            var config = new CLIConfiguration().Map<Command1, MockCommand1Handler>()
```

Comment:
Method and class still is named `RootComandHandlerTest` even `RootCommandHandler` no longer exist

Reply:
Fixed. Agreed.